src/app/shared/http-services/auth.http-service.ts

```diff
@@ -28,16 +28,35 @@ export class AuthHttpService {
     cookie_secure: this.apiUrl.protocol === 'https:' || this.apiUrl.hostname === 'localhost' ? '1' : '0',
     cookie_same_site: '1',
     /**
-     * NOTE on cookie_same_site:
-     * The 'same-site' policy only applies for auth by third-party cookie.
-     * A cookie party is defined by its domain:
-     * - first-party cookies have domain "dev.algorea.org", "api.algorea.org" or "algorea.org"
-     * - third-party cookies have domain or "complete-other-site.com"
+     * NOTE on cookie_same_site and the cookie attribute "SameSite":
      * Safari and Firefox both disable third-party cookies by default, and Chrome will in 2021 (but postponed for 2023)
      * Since third-party cookies will be removed in a near future, we chose not to enable this type of authentication,
      * making the 'same-site' policy always true.
+     *
+     * NOTE on cookie party:
+     * A cookie party is determined by its "Domain" attribute, which defaults to the hostname.
+     * As defined by the MDN glossary, the domain represents an "authority", an organization to which belongs the cookie.
+     * In our case, the authority is "algorea.org".
```

Comment:
It is more "for instance" than "In our case", this domain may change for this app.